### PR TITLE
Observing Squad oriented PR

### PR DESCRIPTION
**Pull request by valentin-lup**

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -57,11 +54,57 @@ case "$1" in
...
+         INDEX=$(( $i - 1 ))
+         WORKDIR="$CUSTOM_HOME/elrond-nodes/node-$INDEX"
+         install
+         install_utils
+         node_name
+         observer_keys
+         systemd
+         if [[ "$INDEX" == 3 ]]; then
+                       sed -i 's/DestinationShardAsObserver = "disabled"/DestinationShardAsObserver = "metachain"/' $WORKDIR/config/prefs.toml
+                       sed -i '/\[DbLookupExtensions\]/!b;n;c\\tEnabled = true' $WORKDIR/config/config.toml
```
Comment (iulianpascalau) [inline script.sh:90]: Duplicated code with L93. Can we take it out from the `if` branch?

File: script.sh [hunk 2]
```diff
@@ -302,16 +399,25 @@ case "$1" in
...
+  echo -e "${GREEN} 8) ${CYAN}github_pull${GREEN} - Get latest version of scripts from github (with variables backup)${NC}"
+  echo -e "${GREEN} 9) ${CYAN}get_logs${GREEN} - Get the logs from all the nodes${NC}"
   echo -e
   ;;
 
-*)
-  echo "Usage: Missing parameter ! [install|upgrade|start|stop|cleanup|get_logs|github_pull|version]"
-  ;;
-esac
+esac
```
Comment (iulianpascalau) [inline script.sh:423]: add empty line

File: script.sh (hunk 1 above)
Comment (iulianpascalau) [reply script.sh:90]: done

File: script.sh (hunk 2 above)
Comment (iulianpascalau) [reply script.sh]: done

File: config/functions.cfg [hunk 3]
```diff
@@ -263,6 +286,84 @@ function keys {
...
+function proxy_config {
+  #Customize the config.toml file for the elrond-proxy
+  echo -e
+  echo -e "${GREEN}Customizing the elrond-proxy config.toml file...${NC}"
+  echo -e
+  
+  sed -i "s/ServerPort = 8080/ServerPort = 8079/g" $CUSTOM_HOME/elrond-proxy/config/config.toml
+  sed -i -n '/4294967295/q;p' $CUSTOM_HOME/elrond-proxy/config/config.toml
+  
+  echo "# shard id 4294967295
```
Comment (andreibancioiu) [inline config/functions.cfg:348]: Not really sure why the comment `# ...`

**Review by andreibancioiu — APPROVED** [review]

File: config/functions.cfg (hunk 3 above)
Comment (iulianpascalau) [reply config/functions.cfg:348]: probably the file was copied as a whole

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**